src/routes/safe/components/Balances/Coins/index.tsx

```diff
@@ -79,6 +83,17 @@ const Coins = (props: Props): React.ReactElement => {
   const safeTokens = useSelector(extendedSafeTokensSelector)
   const granted = useSelector(grantedSelector)
 
+  useEffect(() => {
+    trackEvent(NAVIGATION_EVENTS.BALANCE)
+  }, [])
+
+  const tokenAmount = useMemo(() => safeTokens.size, [safeTokens])
+  useEffect(() => {
+    if (tokenAmount > 0) {
+      trackEvent({ ...ASSETS_EVENTS.TOKEN_AMOUNT, label: tokenAmount })
```

Comment:
This is... controversial. There'll be tons of events with very different amounts.

Reply:
I did raise some concerns about this originally. @johannesmoormann, it would be helpful to debrief on this.

Reply:
After a discussion: in future we can limit it to once per session, but it's fine for now.

Reply:
There's a few events that are over-tracking. I'm going to look into these now.